avocado/utils/cpu.py

```diff
@@ -212,14 +214,17 @@ def get_cpuidle_state():
     :rtype: Dict of dicts
     """
     cpus_list = cpu_online_list()
-    states = range(len(glob.glob("/sys/devices/system/cpu/cpu0/cpuidle/state*")))
+    states = range(
```

Comment:
I think all these indentation changes should not be in this PR. Keep only the new function.

Reply:
@cacarrara ok sure  but this PR has two commits  one for pep8 changes that for indentation changes and other is for new function , is that not ok ? will raise in two PR

Reply:
It's weird, because as far as I could see, our lint check is not failing for these lines. So, seems they're correct taking our current validation rules.

Reply:
@cacarrara yes i know i just run pep8 and give me warning  about these  so thought we can make pep8 complaint , if you think it not needed will re spin the patch  having only new added functionality please suggest

Reply:
I would prefer keep the rules of our current `make check` (that run inspektor lint). We can discuss improvements on our lint and apply them wide in project after.

Reply:
@cacarrara sure make sense